releases/release-7.6.0.md

```diff
@@ -0,0 +1,356 @@
+
+    更多信息，请参考[用户文档](链接)。
+
+* 支持下推字符串函数 `LOWER` 到 TiKV [#48170](https://github.com/pingcap/tidb/issues/48170) @[gengliqi](https://github.com/gengliqi) **tw@qiancai** <!--1607-->
+
+    * `LOWER`
+
+    更多信息，请参考[用户文档](/functions-and-operators/expressions-pushed-down.md)。
+
+* 新增支持下推以下 JSON 函数到 TiFlash [#48350](https://github.com/pingcap/tidb/issues/48350) [#48986](https://github.com/pingcap/tidb/issues/48986) [#48994](https://github.com/pingcap/tidb/issues/48994) [#49345](https://github.com/pingcap/tidb/issues/49345) @[SeaRise](https://github.com/SeaRise) @[yibin87](https://github.com/yibin87) **tw@qiancai** <!--1608-->
+
+    * `JSON_UNQUOTE`
+    * `JSON_ARRAY`
+    * `JSON_DEPTH`
+
+    更多信息，请参考[用户文档](/tiflash/tiflash-supported-pushdown-calculations.md)。
+
+* 包含分区表的执行计划可以被缓存 [#issue号](链接) @[mjonss](https://github.com/mjonss) **tw@qiancai** <!--1662-->
+
+    执行计划缓存是提升交易系统性能的有效手段。 自 7.6.0 开始， TiDB 解除了分区表的执行计划无法进入执行计划缓存的限制， 包含分区表的 SQL 语句能够从执行计划缓存中受益。 这将进一步提升分区表在 TiDB 的应用场景， 客户可以更多的利用分区技术降低数据读取的数量，提升数据库性能。
+
+    更多信息，请参考[用户文档](/sql-prepared-plan-cache.md)。
+
+* 批量建表的性能提升 10 倍 [#issue号](链接) @[gmhdbjd](https://github.com/gmhdbjd) **tw@hfxsd** <!--1408-->
+
+    在之前的版本里，用户将上游数据库上万张表迁移到 TiDB 时， TiDB 需要消耗较长的时间来创建这些表，效率较低。在 7.6 版本引入新的 DDL 架构，可通过系统参数 tidb_ddl_v2 开启，新版本的 DDL 相比之前版本的 DDL 在批量建表的性能有 10 倍的提升，可大大减少用户建表的时间。
+
+    更多信息，请参考[用户文档](链接)。
+
+* 优化器增强对多值索引的支持 [#issue号](链接) @[Arenatlx](https://github.com/Arenatlx) @[time-and-fate](https://github.com/time-and-fate) **tw@ran-huang** <!--1405/1584-->
+
+    TiDB 自 v6.6.0 开始引入[多值索引](/sql-statements/sql-statement-create-index.md#多值索引)，提升对 JSON 数据类型的检索性能。在 v7.6.0，优化器增强了对多值索引的支持能力，在复杂使用场景中，能够正确识别和利用多值索引对查询进行优化。
+
+    * 多值索引上的统计信息会被收集，并应用于优化器估算，当一条 SQL 可能选择到数个多值索引时，能够识别开销更小的索引。
+    * 当出现用 OR 连接的多个 "member of" 条件时，优化器能够利用 `Index Merge` 操作做更高效的条件过滤。
+
+    更多信息，请参考[用户文档](/sql-statements/sql-statement-create-index.md#多值索引)。
+
+### 稳定性
+
+* 跨数据库绑定执行计划 [#issue号](链接) @[qw4990](https://github.com/qw4990) **tw@Oreoxmt** <!--1613-->
+
+    我们看到越来越多的用户运行 TiDB 支撑其 SaaS 平台。SaaS 平台的一种普遍的建模方式，是把平台上每个租户的数据存入不同的“数据库”，而业务逻辑完全相同。这样我们能看到上百个数据库拥有相同的表和索引定义，运行类似的 SQL 语句。在这种场景下，当我们要对一条 SQL 语句的执行计划进行绑定(SQL Binding)，这条绑定通常对运行在其他数据库的 SQL 也有帮助。
+
+    针对这种应用场景，TiDB 引入了 "通用绑定" (Universal SQL Binding) 进行跨数据库绑定。 用一个执行计划绑定匹配模式相同的 SQL，即使他们运行在不同数据库上。比如，当我们创建了下面绑定，无论 `t1` 和 `t2` 位于哪个数据库，TiDB 都会尝试用此绑定的规则来生成执行计划，而不需要为每个数据库上的 SQL 单独创建。
+
+    ```sql
+    CREATE GLOBAL UNIVERSAL BINDING for
+        SELECT * FROM t1, t2 WHERE t1.id = t2.id
+    USING
+        SELECT /*+ merge_join(t1, t2) */ * FROM t1, t2 WHERE t1.id = t2.id;
+    ```
+
+    "通用绑定" 还能够有效缓解 Saas 场景中一个常见问题。SaaS 客户的数据及负载可能因为客户业务本身的需要而剧烈变化，通常 SaaS 服务商无法预测。在这个场景下，由于统计信息无法及时更新，数据量由小及大的快速变化经常会引发 SQL 性能问题。为解决这个问题，SaaS 服务商可以用 "通用绑定" 固定大数据量客户已经验证过的执行计划，所有客户的执行计划都会被固定。 针对 SaaS 平台，这一功能的引入为客户带来了显著的便利和体验提升。
+
+    由于 "通用绑定" 会引入一个很小的系统开销（小于1%）， 默认将其关闭。 有需要的用户通过设置变量 [`tidb_opt_enable_universal_binding`]() 为 `YES` 开启。
+
+    更多信息，请参考[用户文档](/sql-plan-management.md)。
+
```

Comment:
```suggestion
* 闪回功能支持精确 TSO [#48372](https://github.com/pingcap/tidb/issues/48372) @[BornChanger](https://github.com/BornChanger/BornChanger) <!--**[tw@qiancai](mailto:tw@qiancai)** 1615-->

    TiDB v7.5.0 提供了更加强大和精确的闪回功能，不仅支持回溯到过去指定的时间点，还可以精确的指定 [TSO](https://docs.pingcap.com/tidb/stable/tso) 时间戳，实现更加灵活的数据恢复。例如，此功能可和 TiCDC 联合使用，允许下游 TiDB 集群在暂停数据同步、开启预上线读写测试后，优雅快速地回溯到暂停的 TSO 时间戳并继续通过 TiCDC 同步数据，简化了预上线验证流程和数据管理。
    
    你可以通过 `SELECT ... FROM ... AS OF TIMESTAMP` 语句查询指定的 [TSO](https://docs.pingcap.com/tidb/stable/tso) 时间戳的数据，验证后通过 `FLASHBACK CLUSTER` 语句回到指定 [TSO](https://docs.pingcap.com/tidb/stable/tso) 。

    ```sql
    SELECT * FROM users AS OF TIMESTAMP '445494839813079041';
    FLASHBACK CLUSTER TO TIMESTAMP '445494839813079041';
    ````
```